src/NLog/SetupExtensionsBuilderExtensions.cs

```diff
@@ -142,9 +144,18 @@ public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensio
         /// <param name="setupBuilder">Fluent interface parameter.</param>
         /// <param name="name">Name of the layout renderer - without ${}.</param>
         /// <param name="layoutMethod">Callback that returns the value for the layout renderer.</param>
-        public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensionsBuilder setupBuilder, string name, Func<LogEventInfo, LoggingConfiguration, object> layoutMethod)
+        /// <param name="options">Options of the layout renderer.</param>
+        public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensionsBuilder setupBuilder, string name, Func<LogEventInfo, LoggingConfiguration, object> layoutMethod, LayoutRenderOptions options = LayoutRenderOptions.None)
```

Comment:
Think this is a good change. But can it be moved to another PR? And by adding new methods with extra parameter. Since changing existing is breaking change.

Reply:
How can I move it to another PR? It seems that when I push my local commit to Github, it's added to this PR automatically.

Reply:
Right now you are making a PullRequest from your own local `dev`-branch. If you perform `git branch` into another new local-branch, then it can be used for creating another seperate PullRequest from the new local-branch.

Reply:
OK, but how do I remove this commit from current PR? Revert and commit?

Reply:
I'm not very skilled with git. When I need to fix previous commits, then I go into branch history and choose "Reset" and "Keep Changes" (all reverted commits are now in pending changes). Then I make the necessary code-changes and commit again. And then finally perform a `git push -f` (forced). Ofcourse force-push is a no-go when working on shared branches, but it is fine for your own local branches used for pull-requests.